### ISSUE TITLE
More information about gender classification

### ISSUE THREAD

**Issue by Ruchaud** [beniz/deepdetect#85]
Dear,

First, thank for your tool. It is very useful. 

I have some questions about the training of the gender classification. Do you have a paper related to this work?
Because I would like to know which CNN model and which database are you using for the training?

Thanks you in advance.
Regards,
Ruchaud Natacha.

**Comment by beniz** [comment]
Hi Natacha,
the gender model was built in a semi-automated manner by our tools, i.e.:

- the dataset was gathered semi-automatically and among its content are images from many categories in Imagenet
- the dataset was randomly splitted and the model trained from the largest split

We do not publicly release the datasets, for copyright and in this case for obvious privacy reasons. 

There are ways we make some datasets available to our customers, and for this you can get in touch with us, see http://www.deepdetect.com/meta/contact/.

You can also find a bit more details on these model in #33.

**Comment by beniz** [comment]
I had missed that you appear to be from a research institution. You can get in touch, along with your PhD advisor, and we can find ways to accomodate some elements, but privacy issues are always around the corner with this type of models.

**Comment by Ruchaud** [comment]
Hi Beniz,

Thanks you for your answer.
Yes, I am a PhD Student from Eurecom, Sophia-Antipolis, France. I understand that you cannot provide the datasets for privacy reasons. 
I tested your gender classifier on bodies front view and also on bodies profile view. The classification works well on bodies front view but not on bodies profile view. So I just wanted to know if the training dataset contains bodies front view only. But I understand if you cannot provide me the answer.

Best regards,
Ruchaud Natacha.

**Comment by beniz** [comment]
Good to know, can you share the accuracy if you have any metric for this model on data from sources that are obviously outside the initial dataset ? 

One way to improve would be to try to finetune the model on a similar data but with profiles views added. You can easily do so with the `finetune` and `weights` parameters.

**Comment by Ruchaud** [comment]
I tested on 1082 videos from HID gait database (864 male videos and 220 female videos). The accuracy for male detection is 0.996 and the accuracy for female detection is 0.39. The classifier fails to recognize female.

Natacha.

**Comment by beniz** [comment]
I see. What if you finetune on portion of these gait images, and test on a held out test from this dataset ?

**Comment by Ruchaud** [comment]
I have not tested yet. If I do it, I will let you know.
Best, Natacha.
